### PR TITLE
Thank you page #78

### PR DESCRIPTION
**Pull request by matthall8**
Notes: Do not merge before #105 

Updates

- Thank you page designed and styled #78

### CODE REVIEW

File: client/src/pages/CheckoutPage/sections/ThankYou/ThankYou.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import { OrderDescription, H2, OrderDetails, Address } from "./ThankYou.style";
+
+class ThankYou extends Component {
+  render() {
+    return (
+      <React.Fragment>
+        <h1>Thank you for your order</h1>
+        <OrderDescription>
+          <p>
+            Hi Paul, we’ve received your order #1223003 and we are passing it
```
Comment (afifisuha) [inline client/src/pages/CheckoutPage/sections/ThankYou/ThankYou.js]: is that for the dummy data? because it should be 
"hi props.username ... props.meal"

Comment (matthall8) [reply client/src/pages/CheckoutPage/sections/ThankYou/ThankYou.js]: It's a static place holder right now as I'm not sure what you data you guys are holding in the state. We'll update this once we sync with the login / register

Comment (afifisuha) [reply client/src/pages/CheckoutPage/sections/ThankYou/ThankYou.js]: alright)

File: client/src/pages/CheckoutPage/CheckoutPage.style.js [hunk 2]
```diff
@@ -0,0 +1 @@
+import styled from "styled-components";
```
Comment (jema28) [inline client/src/pages/CheckoutPage/CheckoutPage.style.js]: Can remove this if not being used.

**Pull request merged**